depfinder/reports.py

```diff
@@ -82,29 +53,21 @@ def extract_pkg_from_import(name):
 
     Returns
     -------
-
+    most_likely_pkg : str
+        The most likely conda-forge package.
+    import_to_pkg : dict mapping str to sets
+        A dict mapping the import name to a set of possible packages that supply that import.
     """
-    num_letters = _import_map_num_letters()
-    original_name = name
-    while True:
-        try:
-            fllt = name[:min(len(name), num_letters)]
-            import_map = _import_map_cache(fllt)
-            supplying_artifacts = import_map[name]
-        except KeyError:
-            if '.' not in name:
-                return original_name, {}, {}
-            name = name.rsplit('.', 1)[0]
-            pass
-        else:
-            break
-    import_to_artifact = {name: supplying_artifacts}
-    # TODO: launder supplying_pkgs through centrality scoring so we have one thing
-    #  but keep the rest for the more detailed reports
-    supplying_pkgs = {ARTIFACT_TO_PKG[k] for k in supplying_artifacts}
-    import_to_pkg = {name: supplying_pkgs}
-
-    return next(iter(k for k in hubs_auths if k in supplying_pkgs), original_name), import_to_artifact, import_to_pkg
+    from conda_forge_metadata.autotick_bot import map_import_to_package
+    from conda_forge_metadata.libcfgraph import get_libcfgraph_pkgs_for_import
+    try:
+        supplying_pkgs, _ = get_libcfgraph_pkgs_for_import(name)
+        best_import = map_import_to_package(name)
+    except requests.exceptions.HTTPError:
+        supplying_pkgs = set()
```

Comment:
As a user I want to know that there was an issue connecting to the conda-forge metadata for `name`. and also what the stack trace was (or at least the error message). 

can you add an error level warning here in this except block that the conda-forge metadata was unavailable for `name` and also log what the error was? i _think_ `logger.exception` will implicitly add the stack trace.